### DISCUSSION TITLE
Course 1 Lesson 3 - The Copy Problem - Should edit out a clip from Cory Doctorow

### DISCUSSION THREAD

**Discussion by nahuakang** [OpenMined/courses#81] [Course: Our Privacy Opportunity] [answered]
[Lesson 3 The Copy Problem](https://youtu.be/nwbtX9rr_L0) from 9:01 to 9:19, Cory Doctorow talked about the copy problem as a double-edged sword and then decided that he wanted to start over again and proceeded to provide a better explanation.

Expected:
1. Remove the repeated content from 9:01 to 9:19
2. Add the appropriate animation of Cory's name from 9:19 onward for a brief moment.

@mcleonard

**Comment by iamtrask** [comment]
On it!

**Comment by iamtrask** [comment, marked as answer]
Fixed! Thank you so much for going through and pointing these things out Nahua. You rock. New video is in the platform now.
